selftests/unit/test_loader.py

```diff
@@ -50,6 +50,45 @@ def test(self):
     main()
 """
 
+AVOCADO_TEST_TAGS = """#!/usr/bin/env python
+from avocado import Test
+from avocado import main
+
+import time
+
+class DisabledTest(Test):
```

Comment:
We settled on that sometimes it's useful to have the code as snippet, sometimes it's better to have files (`selftests/.data/`) and sometimes we use external files like `example/tests`. For this functionality it'd make sense to me to use existing file as imagine the test fails. How do you debug/adjust/check slightly modified version without the access to those files? It's easier to know you have to run this file and be able to modify some arguments (eg. different args).